### PR TITLE
Support customized branch for committing

### PR DESCRIPTION
**Pull request by aleen42**
PR for #49

### CODE REVIEW

**Comment by lowlighter** [comment]
Hello, thanks for contributing!

Could you also add the option in `/action.yml` under `committer_token` ? Else GitHub will trigger warnings during runs because of an undefined option.

I'll merge it after 👍

**Comment by aleen42** [comment]
Committed

File: action.yml [hunk 1]
```diff
@@ -15,6 +15,11 @@ inputs:
     description: GitHub Token used to commit metrics
     default: ""
 
+  # Branch to commit
+  committer_branch:
+    description: The branch used to commit metrics
+    default: main_branch
```
Comment (lowlighter) [inline action.yml:21]:
```suggestion
    default: ""
```
This is actually the default value passed by GitHub to action, so you need to leave it empty so it'll defaults on `github.context.ref.replace(/^refs[/]heads[/]/, "")` in the action 🙂

**Review by lowlighter — APPROVED** [review]

**Comment by lowlighter** [comment]
Merged it, thanks for contributing 🎉 !

**Comment by aleen42** [comment]
It seems not to work for my repo: [aleen42/aleen42](https://github.com/aleen42/aleen42) and still push to `master` while current branch has been set with `metrics`

**Pull request merged**